docs/vsphere-prerequisites.md

```diff
@@ -1,71 +1,34 @@
-    * `exclude-nics=`
-  * Ensure that the **VMTools** Windows service is running and will
-  start on boot.
-* Pull all the required Windows container base images
-you'll need for your applications. The images you'll pull
-is dependent on the Windows kernel being used. Please see
-[the offical matrix](https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/container-base-images)
-to determine which images you need.
-* Sysprep the VM. Use an `unattend.xml` that will maintain all the
-changes needed for the WMCO. The most important one being that the
-Administrator's home directory stays intact with the SSH public key.
+Follow the provided documentation to configure the vSphere cluster with [hybrid OVN Kubernetes networking with a custom VXLAN port](setup-hybrid-OVNKubernetes-cluster.md#vSphere) to work around the pod-to-pod connectivity between hosts [issue](https://docs.microsoft.com/en-us/virtualization/windowscontainers/kubernetes/common-problems#pod-to-pod-connectivity-between-hosts-is-broken-on-my-kubernetes-cluster-running-on-vsphere).
 
-  Here is an example command to `sysprep`
-  ```
-  C:\Windows\System32\Sysprep\sysprep.exe /generalize /oobe /shutdown /unattend:<path_to_unattend.xml>`
-  ```
-  You can find an example [unattend.xml](unattend.xml) in this repo. It
-  is provided as an example and it **should not** be used directly.
+To create a Windows worker node, you will first need to setup a Windows virtual machine base image (the Golden Image), find enclosed an overview of the process required for the [configuration of a Windows VM golden image in vSphere Platform](vsphere-golden-image.md).
 
-## vSphere environment changes
-WMCO downloads the ignition files from the internal API server
-endpoint. To enable this, add a new DNS entry for
-`api-int.<cluster-name>.domain.com` which points to the external API
-server URL `api.<cluster-name>.domain.com`.
+In addition, a programmatic approach for the creation of the Windows VM golden image can be found [here](vsphere_ci/README.md)
 
-This can be a CNAME or an additional A record.
+The follwing section describes the setup of the internal API server endpoint within the environment.
 
-This new DNS entry ensures that Windows VM can download the ignition
-file from the internal API server and kubelet
-on the configured VM can communicate with the internal API server. In
-the case of Linux nodes, coreDNS is running
-on every node which helps in resolving the internal API server URL. The
-external API endpoint should have been
-created as part of the
-[cluster install](https://docs.openshift.com/container-platform/4.7/installing/installing_vsphere/installing-vsphere-installer-provisioned.html).
+## vSphere environment changes
 
-An example to automate the VM golden image creation can be found
-[here](vsphere_ci/README.md)
+WMCO downloads the ignition files from the internal API server endpoint. To enable this, add a new DNS entry for
+```html
```

Comment:
Doesnt seem like this should be tagged as html

Reply:
agree